WordPress/Classes/ViewRelated/Post/PostListCell.swift

```diff
@@ -95,10 +97,9 @@ final class PostListCell: UITableViewCell, Reusable {
 
         contentView.addSubview(mainStackView)
         contentView.pinSubviewToAllEdges(mainStackView)
-        contentView.backgroundColor = .systemBackground
```

Comment:
We actually need to set the root view's backgroundColor to .systemBackground here. The default background color isn't what we want for dark mode. 

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/5bb9f142-2cc2-4a36-842a-cc17c302e477" width=200>